notebooks/community/model_garden/model_garden_proprietray_image_classification.ipynb

```diff
@@ -0,0 +1,1471 @@
+{
```

Comment:
<p>The model_type seems to be the jist of this tutorial. Yet you set model_type to Cloud?</p><p> </p><p>Also, the migration notebooks are our oldest. I would suggest using a newer AutoML image classification notebook as your base.</p><p> </p><p>The steps need to be updated to reflect that AutoML will train a model garden model.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1792/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>